.travis.yml

```diff
@@ -1,21 +1,24 @@
-language: python
+language: generic
 
-dist: trusty
+sudo: required
 
 services:
-  - postgresql
+  - docker
 
-cache:
-  directories:
-    - $HOME/.cache/pip
+env:
+  - DOCKER_COMPOSE_VERSION=1.21.2
 
-install:
-  - pip install -r requirements.txt
-  - pip install -r requirements_dev.txt
-  - pip install -r requirements_test.txt
+before_install:
+  - sudo rm /usr/local/bin/docker-compose
+  - curl -L https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-`uname -s`-`uname -m` > docker-compose
+  - chmod +x docker-compose
+  - sudo mv docker-compose /usr/local/bin
```

Comment:
We get the docker-compose version we specify, using this `before_install` clause right here.